docs/advanced-backtesting.md

```diff
@@ -141,6 +141,14 @@ Use the `--rejected-signals` option to print out rejected signals.
 freqtrade backtesting-analysis -c <config.json> --rejected-signals
 ```
 
+### Printing analysis on exit signals
+
+Use the `--exit-signals` option to print out analysis on exited signals.
+
+```bash
+freqtrade backtesting-analysis -c <config.json> --exit-signals
+```
```

Comment:
> Since profit has not changed, its good that these numbers remain same :)

That's great! :+1: 

i think we'll need to add this to the docs though - it's not immediately clear i think (something like "this will only impact the output of indicators" or similar).

it also makes the provided running sample (linked to this comment) missleading - as this will output exactly nothing - and suggests that "i can add whatever other parameter i want" - which is technically true - but won't necessarily change the result.

We might even be able to update the output to something more concise:

from 

```
Analysing on exit signals.
Indicators:
┏━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━━━━━━┳━━━━━━━━━━━┳━━━━━━━━━━━┓
┃      pair ┃ enter_reason ┃ exit_reason ┃   emalong ┃  emashort ┃
...
```
to 

```
Indicators on trade exits:
┏━━━━━━━━━━━┳━━━━━━━━━━━━━━┳━━━━━━━━━━━━━┳━━━━━━━━━━━┳━━━━━━━━━━━┓
┃      pair ┃ enter_reason ┃ exit_reason ┃   emalong ┃  emashort ┃
...
```

and obviously the same for the reverse case: something around `Indicators at trade entries:`

(Please treat actual wording as suggestions/hints - it's not necessarily perfect :laughing: )

Reply:
I prefer the "Indicators on trade exits" terminology. Much clearer.